src/main/java/org/apereo/openequella/tools/toolbox/Config.java

```diff
@@ -430,6 +432,42 @@ private void checkConfigsExportItems() {
     }
 
     checkConfig(EXPORT_ITEMS_ATTACHMENT_PATH_TEMPLATE, true, true);
+
+    checkConfig(EXPORT_ITEMS_ITEM_BLACKLIST, true, false);
+    if (validConfig && hasConfig(EXPORT_ITEMS_ITEM_BLACKLIST)) {
+      String[] blacklist = getConfigAsStringArray(EXPORT_ITEMS_ITEM_BLACKLIST);
+      for (int i = 0; i < blacklist.length; i++) {
```

Comment:
is the index `i` directly used?
If not would a `for (String item : list)` makes sense?

Reply:
doh - yeah, that makes more sense. :)